mods/keys.ts

```diff
@@ -0,0 +1,24 @@
+import minimist = require("minimist");
+import { resolve } from "url";
+import fs = require('fs');
+import os = require('os');
+import * as bb from "../lib/bb"
+
+export async function main(cl: bb.Client, args : minimist.ParsedArgs)
+{
+    if (args["_"][1] == "get" && args["name"])
+    {
+        const ret = await bb.GetKeys(cl, args["name"]);
+        console.log(ret);
```

Comment:
add another `if` here for raw text output and map the keys in `ret` to a raw text, one key in each line

Reply:
fixed in 54152d0aebb89e3a0e22387179c1b8ee9e96d618